packages/cli/src/aws.ts

```diff
@@ -0,0 +1,160 @@
+import {
+  CloudFormationClient,
+  DescribeStackResourcesCommand,
+  DescribeStacksCommand,
+  ListStacksCommand,
+  Stack,
+  StackResource,
+  StackSummary,
+} from '@aws-sdk/client-cloudformation';
+import { ECSClient, UpdateServiceCommand } from '@aws-sdk/client-ecs';
+import { Command } from 'commander';
+
+interface MedplumStackDetails {
+  stack: Stack;
+  tag: string;
+  ecsCluster?: StackResource;
+  ecsService?: StackResource;
+  appBucket?: StackResource;
+  storageBucket?: StackResource;
+}
+
+export const aws = new Command('aws').description('Commands to manage AWS resources');
+const client = new CloudFormationClient({});
+const tagKey = 'medplum:environment';
+
+aws
+  .command('list')
+  .description('List Medplum AWS CloudFormation stacks')
+  .action(async () => {
+    const stackSummaries = await listStacks();
+    for (const stackSummary of stackSummaries) {
+      const stackName = stackSummary.StackName;
+      const details = await getStackDetails(stackName);
+      if (!details) {
+        continue;
+      }
+      printStackDetails(details);
+      console.log('');
+    }
+  });
+
+aws
```

Comment:
Another thought could be having the arg `tag` as optional (I believe in commander optional arguments use [] e.g '[tag]'?) And we can see the details of the stack this way too. Up to you

Reply:
Yeah... I considered that, given that the `list` and `describe` commands are currently so similar.  My thinking is that `describe` will evolve over time to show more details, vs `list` will be more of an abbreviated summary.